pkg/utils/k8s/client.go

```diff
@@ -29,9 +24,6 @@ type Client interface {
 	// WithContext returns a client configured to use the provided context on
 	// subsequent requests, instead of one created from the preconfigured timeout.
```

Comment:
we should probably fix that comment since the default context does not have any timeout set